cypress/integration/data-import/mrc-import-with-matching-999ff.spec.js

```diff
@@ -47,7 +48,7 @@ describe('ui-data-import: MARC file import with matching for 999 ff field', () =
       name: mappingProfileNameForExport,
       typeValue : NewFieldMappingProfile.folioRecordTypeValue.instance,
     };
-    SettingsDataImport.goToMappingProfiles();
+    cy.visit(`${settingsMenu.mappingProfilePath}`);
```

Comment:
you don't need to use string template in such cases. You can write string const directly.

Reply:
Done